.github/workflows/ci.yml

```diff
@@ -1,12 +1,12 @@
-name: Ruby
+name: CI
```

Comment:
Is there some kind of de-prefixing going on? We don't have `Ruby / Ruby` at the moment, but this adds the `CI / ` at the beginning 🤔.

I don't really like the `CI / ` prefix.

Reply:
I think it may be something that changed recently about GH actions. #395 that was just opened seems to prefix with `Ruby / Ruby`:
![Screen Shot 2021-04-20 at 1 20 36 PM](https://user-images.githubusercontent.com/22918438/115438275-3429da80-a1db-11eb-8e67-1bdff20f139e.png)

Reply:
If there's no way to omit the name from the Github check (which it doesn't look like there is), IMHO `CI` is an improvement over having `Ruby` twice 😁